release_dashboard/lib/main.dart

```diff
@@ -22,8 +23,10 @@ Future<void> main() async {
     throw Exception('The conductor only supports desktop on MacOS and Linux');
   }
 
+  const bool isDev = true;
```

Comment:
This switch could be made into a button switch widget to toggle between dev and prod in the future like this:
![image](https://user-images.githubusercontent.com/20194490/141157720-673d5c62-4e7d-4a7d-907a-b28995206925.png)